spec.md

```diff
@@ -377,24 +377,31 @@ how Platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Instance. |
-| [update](#input-parameters-object) | object | The schema definition for updating a Service Instance. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Instance. |
+| [update](#json-schema-objects) | object | The schema definition for updating a Service Instance. |
 
 
 ##### Service Binding Object
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Binding. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Binding. |
 
 
-##### Input Parameters Object
+##### JSON Schema Objects
+
+Platforms MAY be able to provide a better user experience if Service Brokers can
+programmatically define the inputs and outputs of requests using JSON Schema
+Object definitions.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| parameters | JSON schema object | The schema definition for the input parameters. Each input parameter is expressed as a property within a JSON object. |
+| parameters | JSON schema object | The schema definition for the input `parameters`. Each input parameter is expressed as a property within a JSON object. |
+| response | JSON schema object | The schema definition for the output response. |
```

Comment:
As part of the /v2/catalog metadata we include the schema for the services and plans. Does it make more sense to include this schema as part of that response too? Additionally, including it here leaves open the possibility that it might change for each message, where if its part of the catalog metadata then its required to be static, at least until the entire catalog is refreshed.

Reply:
Not sure what you mean by "here"? This _is_ the /v2/catalog metadata (specifically `services.plans.schemas`).

Reply:
oh darn, sorry, I scrolled up and say "Service Binding Response" and thought this appeared in the Bind() response json - sorry - ignore my comment.